resources/agents.json

```diff
@@ -0,0 +1,367 @@
+        "background": "linear-gradient(135deg, #00416A 0%, #E4E5E6 100%)",
+        "descriptions": "Specialized in data analysis, visualization, and statistical interpretation.",
+        "tags": ["data", "analysis", "statistics"]
+      },
+      "endpoint": "rag_service",
+      "endpoint_id": "7af2fce4-e6f4-4b57-8832-b5323da7f377",
+      "config": {
+        "system_prompt": "You are a data analyst expert skilled in interpreting data, creating visualizations, and providing statistical insights."
+      },
+      "params": {
+        "temperature": 0.2,
+        "max_tokens": 3000,
+        "top_p": 0.9,
+        "frequency_penalty": 0.1,
+        "presence_penalty": 0.1
+      }
+    },
+    {
+      "id": "d3b07384-d9a1-4d3b-9d3b-07384d9a1d6b",
+      "meta": {
+        "title": "Creative Writer",
+        "avatar": "https://registry.npmmirror.com/@lobehub/assets-emoji/1.3.0/files/assets/angry-face.webp",
+        "background": "#faf3e0",
+        "descriptions": "Specialized in creative writing, storytelling, and content creation with a focus on engaging narratives.",
+        "tags": ["writing", "creative", "storytelling"]
+      },
+      "endpoint": "mistral_small_24b",
+      "endpoint_id": "1037a476-fedc-4494-8d3a-fb69383cf4b9",
+      "config": {
+        "system_prompt": "You are a creative writer skilled in crafting engaging stories, articles, and various forms of content."
+      },
+      "params": {
+        "temperature": 0.9,
+        "max_tokens": 4000,
+        "top_p": 1.0,
+        "frequency_penalty": 0.5,
+        "presence_penalty": 0.5
+      }
+    },
+    {
+      "id": "d3b07384-d9a1-4d3b-9d3b-07384d9a1d7b",
+      "meta": {
+        "title": "Math Tutor",
+        "avatar": "https://registry.npmmirror.com/@lobehub/assets-emoji/1.3.0/files/assets/mechanical-arm.webp",
+        "background": "#e8f4f8",
+        "descriptions": "Expert in mathematics education, from basic arithmetic to advanced calculus and statistics.",
+        "tags": ["mathematics", "education", "tutoring"]
+      },
+      "endpoint": "gemma-7b-it-hkmoon10",
+      "endpoint_id": "e1cbe835-7ccb-4006-90b1-396a2900aea8",
+      "config": {
+        "system_prompt": "You are a patient and knowledgeable mathematics tutor capable of explaining complex concepts in simple terms."
+      },
+      "params": {
+        "temperature": 0.3,
+        "max_tokens": 2000,
+        "top_p": 0.9,
+        "frequency_penalty": 0.0,
+        "presence_penalty": 0.0
+      }
+    },
+    {
+      "id": "d3b07384-d9a1-4d3b-9d3b-07384d9a1d8b",
+      "meta": {
+        "title": "Research Assistant",
+        "avatar": "https://registry.npmmirror.com/@lobehub/assets-emoji/1.3.0/files/assets/brain.webp",
+        "background": "#f5f5f5",
+        "descriptions": "Assists with academic research, literature review, and scholarly writing across various disciplines.",
+        "tags": ["research", "academic", "analysis"]
+      },
+      "endpoint": "gpt-4",
```

Comment:
without `endpoint_id` I got an error below when I click the agent.

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/f20b5a5a-3538-4026-8e1a-e6f2bddf9abe.png)

Reply:
Fixed with updating endpoint_id in required option